injective/assetlist.json

```diff
@@ -737,7 +737,7 @@
           }
         }
       ],
-      "type_asset": "sdk.coin"
+      "type_asset": "cw20"
```

Comment:
This asset doesn't have the cw20: prefix on the base denom, so it cannot be a cw20 (unless it was originally registered incorrectly, and the base denom actually should have the cw20: prefix on it).